tests/dce-compose-test/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3"
+
+services:
+  sonarqube:
+    image: sonarqube:8-datacenter-app
+    depends_on:
+      - db
+      - search
+    deploy:
+      replicas: 1
+    cpus: 0.5
+    mem_limit: 4096M
+    mem_reservation: 1024M
+    ports: 
+      - 9000:9000
```

Comment:
Use a variable for the port to be able to adjust it with line 7 of the test script.